setup.py

```diff
@@ -301,7 +301,7 @@ def build_extension(self, ext):
     cmdclass=cmdclass,
     # scripts=['warpx_1d', 'warpx_2d', 'warpx_rz', 'warpx_3d'],
     zip_safe=False,
-    python_requires=">=3.8",
+    python_requires=">=3.8",  # left for CI, truly ">=3.9"
```

Comment:
Does this comment mean the CI uses Python 3.8?

Reply:
Yes, a few CI runners still use Python 3.8 and for me that is ok for now.
We can bump this once we update CI runners for 3.9+.